openpype/hosts/maya/api/lib.py

```diff
@@ -113,6 +113,8 @@
 
 RENDERLIKE_INSTANCE_FAMILIES = ["rendering", "vrayscene"]
 
+DISPLAY_LIGHTS = ["default", "all", "selected", "active", "none"]
+
```

Comment:
The only question I have is this. That **after** this PR the instance will always override the review Display Lights to `default` by default no matter what. As such, any setting in the Project Settings for Maya for Review is completely ignored.

Does it make more sense to EITHER:
1. Have the creator on create set the default value as defined in the Settings
OR
2. Add a first entry in the enum like "use from project settings" and then convert the index on collect accordingly.

My personal preference might be 1) so that if project settings change that the behavior of an existing scene doesn't suddenly change.

Reply:
Nothing about display lights are currently exposed in the OP settings, so do you mean that we should enforce the display lights to revert to `default` on create?

Reply:
> Nothing about display lights are currently exposed in the OP settings,

Are you sure? I'm seeing this `project_settings/maya/publish/ExtractPlayblast/capture_preset/Viewport Options/displayLights` 

![afbeelding](https://user-images.githubusercontent.com/2439881/207870639-3297f356-0b4b-49d7-854a-0c8198b7f342.png)

Could be that I've merged a PR that on `develop` isn't merged yet.

Reply:
Ahh, didnt see that. Seems a bit confusing to use the settings from the extractor plugin and apply to the creator. Should really be under `project_settings/maya/create/CreateReview`, or am I getting something wrong here?

Reply:
I guess, yes - not sure about the thought process there. Moving it would influence existing productions so I'm not sure how to go about that. It'd be a separate issue to discuss where that should be I suppose.

Reply:
@antirotor got any direction suggestion here?

Reply:
I think it should be in extractor settings and not in review creator just because we might want to decouple playblast and review in the future (for example using some different method than maya playblast to generate review files, like different renderers that might not have the same options.

For now, would it make sense to have option on CreateReview like "Use Settings from extract playblast"?